ENV_VARS.md

```diff
@@ -21,9 +21,95 @@ file.
+base URL (for example, exposed by Docker with no TLS internally behind a
+firewall): http://tiler:3201/. Likewise the OpenAPI documentation, again as an
+example, will be available at http://tiler:3201/docs.
+
+Different setups are possible, and may be preferable over the one above
+depending on specific deployment scenarios.
+
+The `root_path` mechanism when running behind reverse proxies is described in
+great detail in the [FastAPI
+documentation](https://fastapi.tiangolo.com/advanced/behind-a-proxy/): when
+setting up the Foodscapes services behind a reverse proxy, please make sure to
+familiarise yourself with it.
+
+## Datasette service
+
+- `DATASETTE_SERVICE_PORT` (number, required, default is 3203): the port exposed
+  by Docker for the Datasette service; when running an instance under Docker
+  Compose, the Datasette service will always be listening on port 3000
+  internally, and this is mapped to `DATASETTE_SERVICE_PORT` when exposed
+  outside of the container.
+- `DATASETTE_SQLITE_DB_FILENAME` (string, optional, default is 'foodscapes.db'):
+  the filename part of the absolute file path where the core SQLite data file
+  used by the Datasette service can be read from; the full path will is
+  assembled by prepending the path of the folder on the container's local
+  filesystem where the file will be `ADD`-ed during the build of the container
+  image and where Datasette will read it when processing requests.
+
+## Source data
+
+- `DATA_CORE_COG_SOURCE_URL` (URL, required): the URL from which the Docker
+  image build process can fetch the source COG file for the TiTiler service;
+  this needs to be an HTTPS source with no authentication required, such as an
+  URL for an asset on a public S3 bucket.
+- `DATA_CORE_COG_CHECKSUM` (string, required): the checksum (see [Checksums for
+  source data](#checksums-for-source-data) section below for details) of the
+  file above.
+- `DATA_CORE_SQLITE_DB_SOURCE_URL` (URL, required):t he URL from which the
+  Docker image build process can fetch the source SQLite file for the Foodscapes
+  tabular data served by the Datasette service; this needs to be an HTTPS source
+  with no authentication required, such as an URL for an asset on a public S3
+  bucket.
+- `DATA_CORE_SQLITE_DB_CHECKSUM` (string, required): the checksum (see
+  [Checksums for source data](#checksums-for-source-data) section below for
+  details) of the file above.
+
+### Checksums for source data
+
+COG and tabular data are added to the TiTiler and Datasette service container
+images, respectively, fetching them from a remote location through an `ADD`
+Docker build step.
+
+In order to make sure that the files being added are the ones expected (i.e.
+they are the exact revision of the data that is needed, they have not been
+tampered with - either accidentally or maliciously, etc.), a checksum is
+calculated by the Docker build process and this is compared to the checksum
```

Comment:
love this

Reply:
you can see why I wear tin foil hats all the time 😝